package.json

```diff
@@ -1,36 +1,43 @@
 {
     "name": "@prezly/sdk",
-    "version": "1.0.0",
+    "version": "1.0.0-2",
```

Comment:
That's exactly why I wrote the following in the story :wink: 

> Publish as `@prezly/sdk` - start with a `1.0.0-beta.0`/`1.0.0-rc.0` version, publish it, test it as a dependency in another project, and once you're sure it works, you can publish it as `1.0.0`

Reply:
Another trick I use -- is to install `master` directly and test the package before releasing a version.

Reply:
@e1himself you could use [`npm link`](https://docs.npmjs.com/cli/link)

It's exactly the same and doesn't require you to have git.

1. Run `npm link` in source project, in this case it's `prezly/javascript-sdk`
2. Run `npm link [package]`, in this case `npm link @prezly/sdk`

The first will regirster a global link on your device. The second will install from your local device based on the link. It's symlink taken to the level of npm packages.

Reply:
> That's exactly why I wrote the following in the story

@kamilmielnik I don't understand, is it because it took me 3 versions to get it right? 😄Or did you have something else in mind?

Reply:
Well, yeah :smile: 

I hoped that when this story is complete the version will be `1.0.0`, not `1.0.0-2` . 
But :man_shrugging: whatever :smile:

Reply:
@kamilmielnik yes, it will be. This is the "premajor" release. 

![image](https://user-images.githubusercontent.com/6675751/69338775-2f6a9b00-0c64-11ea-879f-c29504480232.png)